<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSyncWizard</class>
 <widget class="QWizardPage" name="AddSyncWizard">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>694</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Grive2 settings</string>
  </property>
  <property name="title">
   <string>Grive2 control panel</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="documentMode">
        <bool>false</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabMessage">
        <attribute name="title">
         <string>message</string>
        </attribute>
        <widget class="QPushButton" name="msgButtonOK">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>350</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
        <widget class="QLabel" name="messageText">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>611</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabSyncData">
        <attribute name="title">
         <string>add sync</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>130</y>
           <width>311</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Select directory</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="syncPath">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>150</y>
           <width>331</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="folderSelButton">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>150</y>
           <width>31</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QPushButton" name="authButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>340</y>
           <width>251</width>
           <height>61</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Google authorization</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="manualSync">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>190</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> Use manual sync</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="scanPeriod">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>220</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
         <property name="maximum">
          <number>6000</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>230</y>
           <width>261</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Google drive sync period</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelAccountName">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>641</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Droid Sans</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>No account present</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="removeSettButton">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>290</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove settings</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveSettButton">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>290</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Save settings</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="disableNotify">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>190</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Disable notifications</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabWeb">
        <attribute name="title">
         <string>web</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QWebView" name="webView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="url">
            <url>
             <string>about:blank</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="questionTab">
        <attribute name="title">
         <string>question</string>
        </attribute>
        <widget class="QPushButton" name="questButtonOK">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>350</y>
           <width>80</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
        <widget class="QLabel" name="questText">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>611</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="questButtonCancel">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>350</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabDriveFileList">
        <attribute name="title">
         <string>drive file list</string>
        </attribute>
        <widget class="QTreeWidget" name="treeFileList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>661</width>
           <height>461</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <column>
          <property name="text">
           <string>Files on Google drive</string>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>661</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Select files that you would like to sync from Google to your disk drive</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveTreeButton">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>530</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Save changes</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabProgress">
        <attribute name="title">
         <string>progress</string>
        </attribute>
        <widget class="QLabel" name="progressLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>220</y>
           <width>651</width>
           <height>141</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
